pyshtools/expand/spharm_functions.py

```diff
@@ -218,7 +218,7 @@ def spharm_lm(l, m, theta, phi, normalization='4pi', kind='real', csphase=1,
         Condon-Shortley phase of (-1)^m will be appended to the spherical
         harmonic functions.
     degrees : optional, bool, default = True
-        If True, colat and phi are expressed in degrees.
+        If True, 'theta' and 'phi' are expressed in degrees.
```

Comment:
can you use back ticks for consistence (and also for use with the web documentation)?

Reply:
Ah, I missed that. Thanks!